### DISCUSSION TITLE
Option to set `relativenumber` in `NeogitStatus` tab?

### DISCUSSION THREAD

**Discussion by mrcjkb** [NeogitOrg/neogit#820] [Q&A] [answered]
Hey :smile:

I have tried setting up an autocommand to `set relativenumber` when in the `NeogitStatus` tab.
So far, I've tried `FileType` with `NeogitStatus` as the pattern, and `TabEnter`, `BufEnter`, `BufNew`, to no avail.

`BufNew` works the first time I open it, but subsequent calls to `neogit.open()` result in the number column disappearing again.

Is there a config option that lets me set `relativenumber` in the `NeogitStatus` window?

**Comment by CKolkey** [comment, marked as answer]
Every time a buffer is loaded, we hide numbers: https://github.com/NeogitOrg/neogit/blob/1b2f888e504dc7911fa43e30c5aa7affcec6f6c7/lua/neogit/lib/buffer.lua#L253-L254

You could wrap that in a config value and expose it to the user. Just keep the defaults as-is :)

**Reply by mrcjkb** [reply]
Thanks 🙏
I'll look into that.

**Comment by dhoizner** [comment]
@mrcjkb is this working for you after your PR? I'm setting `disable_line_numbers` to false, but line numbers are not enabled when i open NeogitStatus

thanks!

**Reply by mrcjkb** [reply]
Nope. Looks like it no longer works for me.
I kind of gave up on using relative line numbers, because I found them useless in neogit (the number keys change the fold level, so I can't use them as prefixes for jumps).

**Reply by dhoizner** [reply]
> Nope. Looks like it no longer works for me.
> 
> I kind of gave up on using relative line numbers, because I found them useless in neogit (the number keys change the fold level, so I can't use them as prefixes for jumps).

Hmm, I wonder what's going on. 

Ah, good point though- thanks for the reply!
